apis/Google.Cloud.Datastore.V1/Google.Cloud.Datastore.V1/Google.Cloud.Datastore.V1.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <Version>3.0.0-alpha00</Version>
+    <Version>3.0.0-beta01</Version>
```

Comment:
I'm unclear why `Version` here has moved from `alpha00` to `beta01`, whereas others (e.g. `Google.Cloud.Debugger.V2`) are still on alpha.

Reply:
Whoops - that one was a mistake. That's because I did a find and replace of 3.0.0-alpha00 to 3.0.0-beta01 for GAX, and accidentally caught this too. Will fix in an extra commit.